tedana/io.py

```diff
@@ -201,7 +205,6 @@ def writeresults(ts, mask, comptable, mmix, n_vols, ref_img):
     ======================    =================================================
     Filename                  Content
     ======================    =================================================
-    ts_OC.nii                 Optimally combined 4D time series.
```

Comment:
Maybe I missed it; why are we getting rid of `ts_OC`?

Reply:
That is an accidental bleedover from #499, which I was trying to split up into more manageable pieces. Reverting that and the removal of `ts_nogs` from `gscontrol.gscontrol_raw` now.

Ultimately we _should_ write out the optimally combined data _way_ earlier and not at the very last step, but that's something that should be addressed elsewhere.

Reply:
Ah, okay, that makes sense.